addons/sms/models/sms_api.py

```diff
@@ -55,4 +55,4 @@ def _send_sms_batch(self, messages):
         params = {
             'messages': messages
         }
-        return self._contact_iap('/iap/sms/1/send', params)
+        return self._contact_iap('/iap/sms/2/send', params)
```

Comment:
Seems like a small change that hides a big one. Why changing the API ? Is "1" outdated ?

Reply:
Yes it will be. Route /2/ can answer back with ode 'unregistered' while in the same situation /1/ will return code 'sms_credit'. We need to keep the behaviour of /1/ for previous versions because we rely on the 'sms_credit' to lead people to register their accounts. Furthermore, Seeing the code as it is, we'd need to backport our modifications to just avoid a crash (e.g. IAP_TO_SMS_STATE['unregistered'] will definitely not work in old versions)